python/lsst/ap/association/packageAlerts.py

```diff
@@ -293,6 +313,7 @@ def produceAlerts(self, alerts, ccdVisitId):
             ccdVisitId of the alerts sent to the alert stream. Used to write
             out alerts which fail to be sent to the alert stream.
         """
+        self._server_check()
```

Comment:
I hope we can eventually drop these and just use the server check on startup, as this adds latency.  But let's leave it for now while we get things running.

Reply:
I think we can afford 2×0.5 seconds, and can always tighten the timeout a little. I'm a bit disappointed that this won't help with interrupted connections, but I'd bet it's still a net gain on expected running time.

Reply:
I'm still working on a solution for interrupted connections, but figured it was important to get this in place for the test/code freeze.